src/Components/UI/Input/InputTypeTwo.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
I can see you are using react v17. This import is not necessary.
https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

Reply:
Thanks for the articles, will update the code.